### PR TITLE
docs: Add some basic information to the getting started section

### PR DESCRIPTION
**Pull request by spydon**
# Description

This updates the getting started documentation to include a starting point for building a game with Flame.

Closes: #3393

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [x] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -44,10 +44,129 @@ There is a set of tutorials that you can follow to get started in the
...
+void main() {
+  runApp(
+    GameWidget(
+      game: FlameGame(),
+    ),
+  );
+}
+```
+
+In Flame we use a concept called the Flame Component System (FCS), which is a way to organize your
```
Comment (luanpotter) [inline doc/README.md:65]: Flame provides

**Review by wolfenrain — APPROVED** [review]

File: doc/README.md [hunk 2]
```diff
@@ -44,10 +44,129 @@ There is a set of tutorials that you can follow to get started in the
...
+game objects in a way that makes it easy to manage them. You can read more about it in the
+[Components](flame/components.md) section.
+
+When you want to start a new game you either have to extend the `FlameGame` class or the `World`
+class. The `FlameGame` is the root of your game and is responsible for managing the game loop and
+the components. The `World` class is a component that can be used to create a world in your game.
+
+So if we want to create a simple game we can do something like this:
```
Comment (luanpotter) [inline doc/README.md:73]: you, we ? should standardize person

File: doc/README.md [hunk 3]
```diff
@@ -44,10 +44,129 @@ There is a set of tutorials that you can follow to get started in the
...
+(see the [Assets Directory Structure](flame/structure.md)) and you have to add it to the
+[assets section](https://docs.flutter.dev/ui/assets/assets-and-images) of your `pubspec.yaml` file.
+In this class we also set the size of the component to 200x200 and the [anchor](flame/components.md#anchor)
+to the center of the component by sending them to the `super` constructor. We also let the user of
+the `Player` class set the position of the component when creating it
+(`Player(position: Vector2(0, 0))`).
+
+To handle input on a component you can add any of our [input mixins](flame/inputs/inputs.md) to the
+component. For example, if you want to handle tap input you can add the `TapCallbacks` mixin to the
+player if you want to handle tap input within the bounds of the players component, of if you want to
```
Comment (luanpotter) [inline doc/README.md:136]: mixed up sentences here

**Review by ufrshubham — APPROVED** [review]

**Review by luanpotter — APPROVED** [review]
LGTM with comments

File: doc/README.md [hunk 4]
```diff
@@ -44,10 +44,129 @@ There is a set of tutorials that you can follow to get started in the
...
+Here is a simple example of how to use the `GameWidget`:
+
+```dart
+import 'package:flame/game.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(
+    GameWidget(
+      game: FlameGame(),
```
[the remark this answers is not in the document]
Comment (spydon) [reply doc/README.md:59]: Since we wanted to highlight that you usually don't have to extend `FlameGame`, but just the `World`, I think this is fine.

**Review by luanpotter — APPROVED** [review]
LGTM

**Pull request merged**